integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -109,10 +109,6 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: status.hostIP
-          - name: FUSE_ALLUXIO_PATH
-            value: {{ .Values.fuse.alluxioPath }}
-          - name: MOUNT_POINT
-            value: {{ .Values.fuse.mountPath }}
```

Comment:
@liugs0213 @maobaolong @Nizifan those environment variables will not set the alluxioPath, mountPoint, requires alluxio.fuse.mount.point and alluxio.fuse.mount.alluxio.path

Reply:
Changes will be reverted, those parameters will still be here to support multiple Fuse yaml mounts/files with one alluxio-configmap.yaml